src/rhsm/config.py

```diff
@@ -199,20 +201,29 @@ def set(self, section, name, value):
                 self.add_section(section)
             super(RhsmConfigParser, self).set(section, name, value)
         if section == "logging" and name == "default_log_level":
-            self.is_log_level_valid(value)
+            if not self.is_log_level_valid(default_log_level) and self.is_log_level_valid(value):
+                print("The default_log_level is being set to a valid value '{value}'.  There should be no further action needed.".format(value=value),
+                      file=sys.stderr)
+            elif self.is_log_level_valid(default_log_level) and not self.is_log_level_valid(value):
+                self.print_invalid_log_level_msg(value)
+            elif not self.is_log_level_valid(default_log_level) and not self.is_log_level_valid(value):
+                print("The default_log_level is still being set to an invalid value '{value}', please try again.".format(value=value),
```

Comment:
I did this for the same reason as above - except this time to let the users the log_level was still invalid.

Reply:
Oh, in this case my note was on the "please try again" part only. Personally speaking, reading "please try again" makes me think that the current command/invocation/etc just failed for a transient error, and I should try it once again _in the same way_. Since it is not the case (an invalid log level will still fail in the same way again and again), then my suggestion was rather on mentioning to set a proper value, rather than "retry".

Reply:
I changed it to state the Log Level has been set to an invalid value.